code/datums/outfits/jobs/command.dm

```diff
@@ -32,15 +30,14 @@
 	shoes = /obj/item/clothing/shoes/brown
 	id_type = /obj/item/weapon/card/id/silver/hop
 	pda_type = /obj/item/device/pda/heads/hop
-	backpack_contents = list(/obj/item/weapon/storage/box/ids = 1)
 
 /decl/hierarchy/outfit/job/secretary
 	name = OUTFIT_JOB_NAME("Command Secretary")
 	l_ear = /obj/item/device/radio/headset/headset_com
 	shoes = /obj/item/clothing/shoes/brown
 	id_type = /obj/item/weapon/card/id/silver/secretary
 	pda_type = /obj/item/device/pda/heads/hop
-	r_hand = /obj/item/weapon/storage/briefcase
+	r_hand = /obj/item/weapon/clipboard
```

Comment:
Should this maybe spawn in the belt slot instead?

Reply:
Why would it?